statsmodels/iolib/foreign.py

```diff
@@ -1160,7 +1160,7 @@ def savetxt(fname, X, names=None, fmt='%.18e', delimiter=' '):
             import gzip
             fh = gzip.open(fname, 'wb')
         else:
-            fh = file(fname, 'w')
+            fh = open(fname, 'w')
```

Comment:
This still misses a ``fh.close()`` at the end of the function it looks like. File should only be closed in some cases. Fixed in ``numpy.savetxt``, so you could copy that.